hooks/boot/20-certs_update.rb

```diff
@@ -18,6 +18,14 @@
   :default => false
 )
 app_option(
+  '--certs-reset',
+  :flag,
+  "This option will reset any custom certificates and use the self-signed CA " \
+  "instead. Note that any clients will need to be updated with the latest " \
+  "katello-ca-consumer RPM.",
```

Comment:
Is it possible to add that info wrt smart-proxy tar file into the info above, so the user knows he has to update his external proxies?